### PR TITLE
TWC 2018

### PR DESCRIPTION
**Pull request by juankristal**
Added wiki page for the TWC 2018.
Added news post for TWC 2018.

Waiting for Ephemeral to get the tournament registration page up and running and WalterToro for review. After that, should be good to go.

### CODE REVIEW

File: wiki/Tournaments/TWC/2018/en.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+| Position | Member |
+| ------------ | -------------- |
+| Management | [![][flag_AR] juankristal](https://osu.ppy.sh/users/443656), [![][flag_DE] p3n](https://osu.ppy.sh/users/123703), [![][flag_ES] Deif](https://osu.ppy.sh/users/318565), [![][flag_US] HappyStick](https://osu.ppy.sh/users/256802), [![][flag_HK] mangomizer](https://osu.ppy.sh/users/1893718), [![][flag_CL] WalterToro](https://osu.ppy.sh/users/5281416) |
+| Map Selectors | [![][flag_HK] mangomizer](https://osu.ppy.sh/users/1893718), [![][flag_DE] Nwolf](https://osu.ppy.sh/users/1910766), [![][flag_DE] Onosakihito](https://osu.ppy.sh/users/290128) |
+| Commentators | _To be decided_ |
+
+## Links
+
+- [Discussion thread](https://new.ppy.sh/forum/t/555072)
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: new?

File: news/2018-03-08-TWC-2018-registrations-open.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "osu!taiko World Cup 2018: Registrations now open!"
+date: 2018-03-08 13:00:00 +0000
+---
+ 
+Once more the drumfest of the osu!taiko World Cup is back for its 8th installment! Some minor tweaks to the ScoreV2 system alongside a fresh map selection team are our premises for an exciting tournament! The Registration Phase already started, no time to waste!
```
Comment (TPGPL) [inline news/2018-03-08-TWC-2018-registrations-open.md]: alongside with
has already started, so there's no time to waste!

File: news/2018-03-08-TWC-2018-registrations-open.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+![](/wiki/shared/news/banners/TWC_2018_banner.jpg)
+ 
+For more details, refer to the **[osu!taiko World Cup 2018 wiki article](/wiki/Tournaments/TWC/2018/)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/713163)**.
+ 
+## Tournament Schedule
+ 
++ Registration Phase: 2018-03-08/2018-03-21
++ Drawings: 2018-03-31 14:00:00 UTC
```
Comment (TPGPL) [inline news/2018-03-08-TWC-2018-registrations-open.md]: Might as well add the 14:00:00 UTC into brackets

File: news/2018-03-08-TWC-2018-registrations-open.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+_The rest of the prizes will be determined during the registration phase._
+ 
+## How do I register?
+ 
++ **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/14)**
++ Do not create teams in advance! Teams will be formed by the country’s team captain after the registration phase has concluded.
++ Hand in your registration before the **21st of March.**
+
+Please note - you may only participate if you are of a **osu!taiko global ranking of #5000 or higher and have not violated the [osu! community rules](/wiki/Rules) in the past 12 months**.
```
Comment (TPGPL) [inline news/2018-03-08-TWC-2018-registrations-open.md]: If you are #5000 in the osu!taiko global ranking or higher

File: news/2018-03-08-TWC-2018-registrations-open.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ 
+If your registration successfully passes our evaluation, you will be put on the candidate list for your country. Whenever a captain for your country has been assigned, the list will be sent to them. This may take a while, so please be patient.
```
Comment (TPGPL) [inline news/2018-03-08-TWC-2018-registrations-open.md]: is assigned

File: wiki/Tournaments/TWC/2018/en.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+tags:
+- TWC 2018
+- TWC2018
+---
+osu!taiko World Cup 2018
+=============================
+
+![osu!taiko World Cup 2018](logo.jpg)
+
+The **osu!taiko World Cup 2017** (**_TWC 2018_**) is a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It is the 8th installment of the osu!taiko World Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: 2017 :thinking:

File: wiki/Tournaments/TWC/2018/en.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| ---: | :--- |
+| Registration Phase | 2018-03-08/2018-03-21 |
+| Drawings | 2018-03-31 14:00:00 UTC |
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: 14:00:00 utc in brackets

File: wiki/Tournaments/TWC/2018/en.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+| --- | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | profile badge, "osu!taiko Champion" user title |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | profile badge |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | profile badge |
+
+_More prizes to be defined_
+
+## Organization
+
+The osu!taiko World Cup 2018 is ran by various community members by distributing the multitude of tasks into various fields of responsibility.
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: Is run

File: wiki/Tournaments/TWC/2018/en.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+| Position | Member |
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: Members

File: wiki/Tournaments/TWC/2018/en.md (hunk 1 above)
Comment (TPGPL) [reply wiki/Tournaments/TWC/2018/en.md]: community/forums/topics

File: news/2018-03-08-TWC-2018-registrations-open.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+## Tournament Schedule
+ 
++ Registration Phase: 2018-03-08/2018-03-21
++ Drawings: 2018-03-31 14:00:00 UTC
++ Group Stage: 2018-04-07/2018-04-08
++ Round of 16: 2018-04-14/2018-04-15
++ Quarterfinals: 2018-04-21/2018-04-22
++ Semifinals: 2018-04-28/2018-04-29
++ Finals - Week 1: 2018-05-05/2018-05-06
++ Finals - Week 2: 2018-05-12/2018-05-13
```
Comment (WalterToro) [inline news/2018-03-08-TWC-2018-registrations-open.md]: I would change this for Finals/Grand Finals

File: wiki/Tournaments/TWC/2018/en.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
+---
+tags:
+- TWC 2018
+- TWC2018
+---
+osu!taiko World Cup 2018
+=============================
+
+![osu!taiko World Cup 2018](logo.jpg)
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: This logo should be replaced with the transparent one

File: wiki/Tournaments/TWC/2018/en.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+| Event | Timestamp |
+| ---: | :--- |
+| Registration Phase | 2018-03-08/2018-03-21 |
+| Drawings | 2018-03-31 14:00:00 UTC |
+| Group Stage | 2018-04-07/2018-04-08 |
+| Round of 16 | 2018-04-14/2018-04-15 |
+| Quarterfinals | 2018-04-21/2018-04-22 |
+| Semifinals | 2018-04-28/2018-04-29 |
+| Finals - Week 1 | 2018-05-05/2018-05-06 |
+| Finals - Week 2 | 2018-05-12/2018-05-13 |
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: Again, Finals/Grand Finals

File: wiki/Tournaments/TWC/2018/en.md [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+**Winning conditions:**
+   - In Group Stage, you need to win 4 beatmaps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarterfinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+   - In Semifinals, you need to win 6 beatmaps to win a match. (Best-of-11)
+   - In the **Finals and Grand Finals**, you need to win 7 beatmaps to win a match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+  - Room settings are osu!taiko, Team-Vs., Win Condition: 'Score'. Room name must be "TWC 2017: (TeamBlue) vs (TeamRed)".
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: Should be the other way around, TeamRed vs TeamBlue

File: wiki/Tournaments/TWC/2018/en.md [hunk 14]
```diff
@@ -0,0 +1,239 @@
...
+  - The team mentioned first in the room name must be the blue team, the team mentioned second in the room name must be the red team.
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: First Red, Second Blue

File: wiki/Tournaments/TWC/2018/en.md [hunk 15]
```diff
@@ -0,0 +1,239 @@
...
+
+2. Players are free to select up to two warm-up maps. Using beatmaps with questionable content is prohibited. All maps must be osu!taiko specific maps.
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: "Each team is free to select up to one warm-up beatmap"

File: wiki/Tournaments/TWC/2018/en.md [hunk 16]
```diff
@@ -0,0 +1,239 @@
...
+3. Each captain can ban **one beatmap** to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.  
+4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
+5. Each captain must use "!roll" once in #multiplayer.
+  - The winner of the !roll starts picking the first beatmap of the match.
+  - The loser of the !roll starts banning.
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: ...followed by the winner of the !roll to ban a beatmap.

File: wiki/Tournaments/TWC/2018/en.md [hunk 17]
```diff
@@ -0,0 +1,239 @@
...
+
+6. Captains may pick freely from any bracket.
+  - In case of a tie, the tiebreaker map must be played.
+
+7. Results of the Group Stage will be published via a Statistics sheet.
+
+### Mappool Instructions
+
+1. There will be 1 mappool for the Group Stage, 1 mappool for the Round of 16, 1 mappool for the Quarterfinals, 1 mappool for the Semifinals and 1 mappool for the Finals.
+  - Finals Week 1 & Finals Week 2 use the same mappool.
```
Comment (WalterToro) [inline wiki/Tournaments/TWC/2018/en.md]: Finals and Grand Finals

File: news/2018-03-08-TWC-2018-registrations-open.md (hunk 2 above)
Comment (mangomizer) [reply news/2018-03-08-TWC-2018-registrations-open.md]: No. "alongside a" is fine. 
"along with" is an alternative, "alongside with" doesn't exist.

File: wiki/Tournaments/TWC/2018/en.md (hunk 16 above)
Comment (juankristal) [reply wiki/Tournaments/TWC/2018/en.md]: dont think this is needed to be honest.

**Comment by Ephemeralis** [comment]
skidaddle skidoodle your pr is now a

fuck

**Pull request merged**